first_assesment/service/pokemon.go

```diff
@@ -0,0 +1,44 @@
+package service
+
+import (
+	"first/model"
+	"first/repository"
+	"fmt"
+)
+
+type PokemonService struct {
+	pokemonRepository *repository.PokemonRepository
+}
+
+func NewPokemonService() (*PokemonService, error) {
+	pokemonRepository, err := repository.NewPokemonRepository()
+	if err != nil {
+		return nil, err
+	}
+
+	return &PokemonService{
+		pokemonRepository: pokemonRepository,
+	}, nil
+}
+
+func (s *PokemonService) Saludo() {
+	fmt.Println("Hello world")
+}
+
+func (s *PokemonService) GetAll() ([]*model.Pokemon, error) {
+	fmt.Println("Enter to get all pokemons!!!")
```

Comment:
So does this line, right?

Although it's good to send info to the screen, you'd better use a log instead of using print for this kind of messages.

Could use standard library `log`, or use a 3rd party package such as `logrus` or `zerolog`.

Reply:
I used Log instead of Println,thanks.